ts/src/bitget.ts

```diff
@@ -2656,6 +2665,12 @@ export default class bitget extends Exchange {
                 request['size'] = this.amountToPrecision (symbol, amount);
                 if (postOnly) {
                     request['timeInForceValue'] = 'post_only';
+                } else if (timeInForce === 'gtc') {
+                    request['timeInForceValue'] = 'normal';
+                } else if (timeInForce === 'fok') {
+                    request['timeInForceValue'] = 'fok';
+                } else if (timeInForce === 'ioc') {
+                    request['timeInForceValue'] = 'ioc';
```

Comment:
@sc0Vu this code is duplicated twice for no reason, we should avoid things like this and dry the code whenever possible